examples/chart/event-handler/templates/configmap.yaml

```diff
@@ -11,6 +11,11 @@ data:
     batch = {{ .Values.eventHandler.batch }}
     window-size = {{ default "24h" .Values.eventHandler.windowSize | quote }}
     debug = {{ default "false" .Values.eventHandler.debug }}
+    {{- if .Values.eventHandler.types }}
+    types = "{{ .Values.eventHandler.types }}"
+    {{- end }}
+    skip-event-types = "{{ default "" .Values.eventHandler.skipEventTypes }}"
```

Comment:
these fields are arrays and we should preserve them either by receiving an array and joining the entries with `,` and quoting or satisfy the toml array

eg1:
```toml
skip-event-types = "evt2,evt2,evet3"
```

or 
```toml
skip-event-types = ["evt2","evt2", "evet3"]
```